client/me/account/style.scss

```diff
@@ -96,6 +96,16 @@
 	font-size: $font-body-small;
 }
 
+.account__hosting-dashboard-notice-banner {
```

Comment:
By default the `<NoticeBanner>` text size is much larger than what is used elsewhere in `/me/account`